app/services/work_version_event_description_builder.rb

```diff
@@ -80,6 +80,8 @@ def license
   end
 
   def files
+    return unless form.input_params
+
     attributes = form.input_params[:attached_files_attributes]
```

Comment:
`form.input_params` seems to be nil on unpersisted form with no files added (ie. new deposit where you just click "save as draft") -- this guards against it